crates/router/src/connector/opennode/transformers.rs

```diff
@@ -211,12 +211,9 @@ impl TryFrom<types::RefundsResponseRouterData<api::RSync, RefundResponse>>
 }
 
 //TODO: Fill the struct with respective fields
-#[derive(Default, Debug, Serialize, Deserialize, PartialEq)]
+#[derive(Debug, Deserialize)]
 pub struct OpennodeErrorResponse {
-    pub status_code: u16,
```

Comment:
Previously we had template code for ErrorResponse?

Reply:
Hey yes it was there, but I was not aware of that to change.